### DISCUSSION TITLE
Singlestat alerts for Prometheus data sources

### DISCUSSION THREAD

**Discussion by thewilli** [grafana/grafana#59927] [Alerting]
As #6983 only led to Singlestat alerts for Azure Kusto and SQL data sources, I‘d like to request this for Prometheus data sources as well

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
